src/app/components/challenge/challengesettings/challengesettings.component.html

```diff
@@ -27,6 +27,9 @@
           </mat-chip-list>
         </mat-form-field>
         <div class="error-message" *ngIf="isValidationError">{{message}}</div>
+        <div class="btn btn-waves-effect waves-dark grad-btn grad-btn-dark fs-14" (click)="reflectChange()">
```

Comment:
Please change this to `updateBannedEmailList`

Reply:
@Sanji515 Done :+1: